galaxy_importer/utils/spdx_licenses.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+# use get_spdx() to ref the spdx_license info, so it
+# only loaded from disk once
+_SPDX_LICENSES = None
+
+
+def load_spdx():
+    cwd = os.path.dirname(os.path.abspath(__file__))
```

Comment:
The best approach would be not to depend on physical location of this particular file and use `pkg_resources` interface for data file lookup. 

Note: From python 3.7 there is a standard package [importlib.resources](https://docs.python.org/3.7/library/importlib.html#module-importlib.resources), but we still support 3.6 I suppose.

Reply:
Updated to use `pkg_resources`